test/README.md

```diff
@@ -65,6 +65,65 @@ Note:
 - `AUTO_UPDATE` doesn't work with tests that have multiple `TEST_OUTPUT` segments
 - `AUTO_UPDATE` can be set as an environment variable or as Makefile-like argument assignment
 
+### Running the Unit Tests
+
+The unit tests will automatically run when all tests are run using `./run.d` or
+`make`. To only run the unit tests the `./run.d unit_tests` command can be used.
+
+For more finer grain control over the unit tests the `./unit_test_runnner.d`
+command can be used:
```

Comment:
I would recommend just using `run.d` as an interface (it can pass all arguments to `unit_test_runner` when an e.g. a `-u` shortcut is used) as this should simplify the exposed tools to the user (which we'll need to maintain) and it will make bootstrapping the binaries easier.

Reply:
I can give that a try.

Reply:
Done.